src/equations/compressible_euler_2d.jl

```diff
@@ -575,6 +575,34 @@ function boundary_condition_sedov_self_gravity(u_inner, orientation, direction,
 end
 
 
+"""
+    boundary_state_free_slip_wall(u_internal, normal_direction::AbstractVector,
```

Comment:
Yeah, it is "free" slip to avoid confusion with a "no" slip wall. We can just call it a slip wall. I am never certain how standard such naming conventions are across scientific disciplines :)

Reply:
I like the symmetry, so let's just keep it